README.md

```diff
@@ -144,6 +144,17 @@ export OPENAI_API_KEY=your_openai_api_key_here
 export OPENAI_ENDPOINT=https://dashscope.aliyuncs.com/compatible-mode/v1
 kubectl-ai --llm-provider=openai --model=qwen-plus
 ```
+
+#### Using AWS Bedrock
```

Comment:
Actually, we have seen more requests about supporting anthropic support their SDK directly. Not much about bedrock honestly :)